index.js

```diff
@@ -81,6 +81,18 @@ const getBrowserifyOptions = (entry, userBrowserifyOptions = {}) => {
     entries: [entry],
   })
 
+  if (typescriptPath) {
+    browserifyOptions.extensions.push('.ts', '.tsx')
+    // remove babelify setting
+    browserifyOptions.transform.pop()
```

Comment:
We can't assume the babelify transform will be the last item. The user could have passed options in with transforms that are different from the default. Instead of use `.pop()`, we should loop through the transforms, see if one matches `babelify`, and slice it out if it exists. 

We might also need to detect if they use the `typescript` option _and_ pass in `tsify` themselves as a transform. In that case, we should probably error and say something like "We see you passed the `typescript` option and also passed a browserify transform for TypeScript. Please only do one or the other."